tests/integration/nm/linux_bridge_test.py

```diff
@@ -268,26 +267,8 @@ def _remove_read_only_properties(bridge_state):
 
 @pytest.fixture
 def nm_unmanaged_dummy1():
-    cmdlib.exec_cmd(
-        f"ip link add name {DUMMY1} type dummy".split(" "), check=True
-    )
-    cmdlib.exec_cmd(f"ip link set {DUMMY1} up".split(" "), check=True)
-    cmdlib.exec_cmd(
-        f"nmcli dev set {DUMMY1} managed yes".split(" "), check=True
-    )
-    time.sleep(1)  # Wait device became managed
-    yield
-    libnmstate.apply(
-        {
-            Interface.KEY: [
-                {
-                    Interface.NAME: DUMMY1,
-                    Interface.STATE: InterfaceState.ABSENT,
-                }
-            ]
-        },
-        verify_change=False,
-    )
+    with nm_unmanaged_dummy(DUMMY1):
+        yield
```

Comment:
Can we use directly `nm_unmanaged_dummy`?

Reply:
The I need to do the `try-finally` stuff which is done by pytest fixture.

Reply:
Oh I see. I am fine with this then.